### PR TITLE
RIA-6555 LR Notification when HO has been directed to submit the bundle

### PR DESCRIPTION
**Pull request by augustio**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-6555

### Change description ###
* Send a modified notification to legal rep on request respondent evidence event for ADA appeal

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -151,7 +151,9 @@ govnotify:
         email: 9c8ab848-b1f7-4dbe-9854-4922e94ece67
         sms: e805b3c5-b1e2-4ad1-9416-3298ca765738
       legalRep:
-        email: 33430912-89d5-416d-8d49-52eae9ce82c3
+        email:
+          ada: f43d0290-c78b-4698-a95e-b967cbd312b7
+          nonAda: 33430912-89d5-416d-8d49-52eae9ce82c3
```
Comment (colmlynch60) [inline src/main/resources/application.yaml:156]: @augustio  the template 33430912-89d5-416d-8d49-52eae9ce82c3 doesn't match template per link in story - 

See the strike out of "Home Office" -
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1191575812&__ncforminfo=suyFC86SYP_oSwJhuzaH6XmdScY6TxJcHuIEMzeZJu-Yl_XPFzP4B2whtcU7jtWOugP4CW0wR3PtLLy6WfFHv2RxZSb8hTNi&__ncforminfo=TeXtm3SEAq5_WgMNyXR3y72-5qTlZSYrc97AAOXtpLNpdyFCb899bop9TuADgTgaX3EkN_CG5PAu7r2gbbONKsR4MoEEHD35bxfwgUJompY=

**Review by colmlynch60 — CHANGES_REQUESTED** [review]

**Review by colmlynch60 — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/AppealService.java [hunk 2]
```diff
@@ -24,4 +25,10 @@ public boolean isAppellantInPersonJourney(AsylumCase asylumCase) {
             .read(AsylumCaseDefinition.JOURNEY_TYPE, JourneyType.class)
             .map(type -> type == JourneyType.AIP).orElse(false);
     }
+
+    public boolean isAdaAppeal(AsylumCase asylumCase) {
```
Comment (ThomasKKC) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/AppealService.java:29]: We have a util class already for this, AsylumCaseUtils, you can reuse

**Review by ThomasKKC — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yaml (hunk 1 above)
Comment (augustio) [reply src/main/resources/application.yaml:156]: This is addressed in a different ticket.

**Review by ThomasKKC — APPROVED** [review]

**Pull request merged**